hawkbit-repository/hawkbit-repository-jpa/src/main/resources/db/migration/H2/V1_12_12__add_action_external_id___H2.sql

```diff
@@ -0,0 +1,2 @@
+    ALTER TABLE sp_action ADD column external_ref VARCHAR(64);
```

Comment:
formatting is a bit messed up here, remove indentation at the beginning, and add a new line at the end of the file (for github)